src/halmos/sevm.py

```diff
@@ -1536,26 +1517,54 @@ def sstore(self, ex: Exec, addr: Any, loc: Any, val: Any) -> None:
 
         self.storage_model.store(ex, addr, loc, val)
 
-    def resolve_address_alias(self, ex: Exec, target: Address) -> Address:
+    def resolve_address_alias(
+        self, ex: Exec, target: Address, stack, step_id
+    ) -> Address:
```

Comment:
i don't like passing the worklist items as additional arguments, but i tried to keep this pr minimal, and plan to refactor this in a future pr.

Reply:
done